src/App.jsx

```diff
@@ -14,33 +18,83 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+const sortGoods = (goods, sortParam, reverseQuery) => {
+  const goodsToSort = [...goods];
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+  if (sortParam) {
+    goodsToSort.sort((item1, item2) => {
+      switch (sortParam) {
+        case SORT_BY_NAME:
+          return item1.localeCompare(item2);
+        case SORT_BY_LENGTH:
+          return item1.length - item2.length;
+        default:
+          return 0;
+      }
+    });
+  }
 
-      <button type="button" className="button is-warning is-light">
-        Reverse
-      </button>
+  if (reverseQuery) {
+    goodsToSort.reverse();
+  }
 
-      <button type="button" className="button is-danger is-light">
-        Reset
-      </button>
-    </div>
+  return goodsToSort;
+};
+
+export const App = () => {
+  const [sortField, setSortField] = useState('');
```

Comment:
The initial state of `sortField` is set to an empty string. This might lead to confusion since it doesn't correspond to any of the defined sort parameters (`SORT_BY_NAME` or `SORT_BY_LENGTH`). Consider setting a default sort parameter or handling the empty state more explicitly.